### ISSUE TITLE
[Server] Unable to launch

### ISSUE THREAD

**Issue by AnnikenYT** [xJon/Tekkit-2#137]
## TL;DR
- **Are you using a legitimate Minecraft account?**: ✅
- **Are you using Java 8, 64-bit?**: ✅
- **Did you add any additional mods**: [Tekkit 2](https://www.technicpack.net/modpack/tekkit-2.1935271) ([Server Pack](https://servers.technicpack.net/Technic/servers/tekkit-2/Tekkit-2_Server_v1.1.2.zip) | [+1](https://www.curseforge.com/minecraft/mc-mods/rope-bridge), no impact on the issue, tried w/ and w/o)
- **Crash/latest log link (if applicable)**: [Full Log](https://mclo.gs/Q4aRqfJ) | [Skip to error](https://mclo.gs/Q4aRqfJ#L296)
- **Issue on Logistics Pipes Repo**: [#1677](https://github.com/RS485/LogisticsPipes/issues/1677)

## Desc.
After updating my [Tekkit 2](https://www.technicpack.net/modpack/tekkit-2.1935271) Server to the newest version, the server couldn't start anymore. Considering that according to the [changelog](https://www.technicpack.net/modpack/tekkit-2.1935271/changelog/2145406), only Logistic Pipes got updated, and configs got changed. `Duplicate interface name "logisticspipes/asm/addinfo/IAddInfoProvider" in class file net/minecraft/item/ItemStack` shows up in the error log, which seems like it is a logistics pipes issue.

**Comment by AnnikenYT** [comment]
Issue fixed, seems like I did not update a dependency or something like that...
